recipe/meta.yaml

```diff
@@ -30,20 +30,16 @@ build:
   track_features:
     - openjdk
 
-requirements:
-  build:
-    - curl  # [unix]
-
 test:
+  requires:
+    - {{ compiler('c') }}
```

Comment:
IDK if selectors work here, but we might want to restrict this to Unix as we don't have a Windows JNI test yet.

Reply:
It looks like the toolchain was added, so the selector worked for the test section.

Reply:
Should I restrict this to linux and osx?